envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/MetadataNodeGroup.kt

```diff
@@ -146,9 +146,8 @@ class MetadataNodeGroup(
     }
 
     private fun hasAllServicesDependencies(metadata: NodeMetadata): Boolean {
-        return !properties.outgoingPermissions.enabled || metadata.proxySettings.outgoing.containsDependencyForService(
-            properties.outgoingPermissions.allServicesDependencies.identifier
-        )
+        return !properties.outgoingPermissions.enabled ||
```

Comment:
why did this condition change?

Reply:
This condition didn't change. I calculate second part of condition in splitToDependencyTypes method